Quaver.Shared/Screens/Result/UI/ResultHitDifferenceGraph.cs

```diff
@@ -162,6 +163,12 @@ private void CreateFakeHitStats()
         /// <exception cref="ArgumentOutOfRangeException"></exception>
         private static ScoreProcessor GetScoreProcessor(ResultScreen screen)
         {
+            if (screen.Gameplay != null && screen.Gameplay.InReplayMode)
+            {
+                var im = screen.Gameplay.Ruleset.InputManager as KeysInputManager;
+                return im?.ReplayInputManager.VirtualPlayer.ScoreProcessor;
+            }
```

Comment:
Doesn't the case at the bottom cover replays already?

Reply:
Yes, but it also reprocesses the replay which isn't necessary anymore.

Reply:
Could you add a comment about that?

Reply:
done